### PR TITLE
Tabbed UI for membership builder

### PR DESCRIPTION
**Pull request by rlafuente**
Some UX details missing:

- [x] include a confirmation hint when update is finished
- [x] review padding for Submit buttons
- [x] the first tab should be open by default when opening the membership builder

Otherwise, the feature should be working as expected.

image

### CODE REVIEW

**Comment by wpf500** [comment]
Just a quick FYI: you should be able to use `AppForm` to handle all the loading/confirmation/error states of the update

File: src/pages/admin/membership-builder.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { useRoute } from 'vue-router';
+
+const { t } = useI18n();
+const route = useRoute();
+
+const tabs: Ref<TabItem[]> = ref([
+  {
+    id: 'adminMembershipBuilderJoinForm',
+    label: t('membershipBuilder.steps.joinForm.title'),
+    to: `/admin/membership-builder/join-form`,
```
Comment (wpf500) [inline src/pages/admin/membership-builder.vue]: Incorrect `to`, gives not found

File: src/pages/admin/membership-builder/index.vue [hunk 2]
```diff
@@ -1,135 +1,166 @@
...
 import { useI18n } from 'vue-i18n';
-import PageTitle from '../../../components/PageTitle.vue';
-import AppStepper from '../../../components/stepper/AppStepper.vue';
-import { Step } from '../../../components/stepper/stepper.interface';
-import EditJoinForm from '../../../components/pages/admin/membership-builder/steps/EditJoinForm.vue';
-import AccountConfirmation from '../../../components/pages/admin/membership-builder/steps/AccountConfirmation.vue';
-import IntroMessage from '../../../components/pages/admin/membership-builder/steps/IntroMessage.vue';
-import Emails from '../../../components/pages/admin/membership-builder/steps/EmailsStep.vue';
-import AppButton from '../../../components/forms/AppButton.vue';
-import { MembershipBuilderEmitter } from '../../../components/pages/admin/membership-builder/membership-builder.interface';
```
Comment (wpf500) [inline src/pages/admin/membership-builder/index.vue:64]: Can you delete this now very happily unused file (membership-builder.interface)?

**Review by wpf500 — CHANGES_REQUESTED** [review]
Good stuff! Just needs a couple of changes and to fix merge conflicts then good to go

File: src/pages/admin/membership-builder.vue [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<route lang="yaml">
+name: adminMembershipBuilder
+meta:
+  pageTitle: membershipBuilder.title
+  role: admin
+</route>
+
+<template>
+  <PageTitle :title="t('membershipBuilder.title')" border />
```
Comment (wpf500) [inline src/pages/admin/membership-builder.vue]: As per the design spec, page titles with horizontal tabs don't have a border. You can remove `border` attribute :)

**Comment by rlafuente** [comment]
Done!

**Review by wpf500 — APPROVED** [review]

**Pull request merged**